plextraktsync/config/HttpCacheConfig.py

```diff
@@ -30,6 +34,52 @@ class HttpCacheConfig:
         "*.trakt.tv/users/*/watchlist/shows": DO_NOT_CACHE,
         "*.trakt.tv/users/likes/lists": DO_NOT_CACHE,
         "*.trakt.tv/users/me": DO_NOT_CACHE,
+
+        # Online Plex patterns
+        "metadata.provider.plex.tv/library/metadata/*/userState": DO_NOT_CACHE,  # XXX bogus?
+        "metadata.provider.plex.tv/library/metadata/*?*includeUserState=1": DO_NOT_CACHE,
+        "metadata.provider.plex.tv/library/metadata/*": LONG_EXPIRY,
```

Comment:
I suggest to simply use `NEVER_EXPIRES` or `timedelta(weeks=x)`

Reply:
will go with timedelta without rand